### PR TITLE
feat: pseudo-translate fixtures created by setup wizard

### PR DESCRIPTION
**Pull request by barredterra**
This PR achieves the following goals:

1. Names of fixture records need to be inserted in english because they are hardcoded in many places
2. Names of fixture records should get translated in the UI
3. Names of fixture records should get extracted as translatable strings

The combination of (1) and (3) is achieved by redefining the translation function to return the input unchanged (only for this file). Hence, the strings won't get translated but the extractor can still pick them up.

(2) is achieved by enabling "Translate Link Fields" for the respective DocTypes.

> no-docs

### CODE REVIEW

File: erpnext/setup/setup_wizard/operations/install_fixtures.py [hunk 1]
```diff
@@ -68,49 +75,49 @@ def install(country=None):
 		# Stock Entry Type
 		{
 			"doctype": "Stock Entry Type",
-			"name": "Material Issue",
+			"name": _("Material Issue"),
```
Comment (cogk) [inline erpnext/setup/setup_wizard/operations/install_fixtures.py:78]: https://github.com/frappe/erpnext/blob/18bc1df6b7bbdfcf3aa526d2687eec2169b26024/erpnext/assets/doctype/asset_repair/asset_repair.py#L211-L213

Comment (cogk) [reply erpnext/setup/setup_wizard/operations/install_fixtures.py:78]: ~~I believe it could be easier to add a `title` field and keep English `name`s?~~

Comment (barredterra) [reply erpnext/setup/setup_wizard/operations/install_fixtures.py:78]: It does stay English, that's the point of this PR. Here, `_` is the identity function, not the translation function.

Comment (cogk) [reply erpnext/setup/setup_wizard/operations/install_fixtures.py:78]: Argh I'm sorry @barredterra, I should have read the description… My primordial brain took control :laughing:

**Pull request merged**